ext/openssl/ossl_cipher.c

```diff
@@ -341,18 +341,18 @@ ossl_cipher_update_long(EVP_CIPHER_CTX *ctx, unsigned char *out, long *out_len_p
     long out_len = 0;
 
     do {
-	int in_part_len = in_len > limit ? limit : (int)in_len;
+        int in_part_len = in_len > limit ? limit : (int)in_len;
 
-	if (!EVP_CipherUpdate(ctx, out ? (out + out_len) : 0,
-			      &out_part_len, in, in_part_len))
-	    return 0;
+        if (!EVP_CipherUpdate(ctx, out ? (out + out_len) : 0,
+                      &out_part_len, in, in_part_len))
+            return 0;
 
-	out_len += out_part_len;
-	in += in_part_len;
+        out_len += out_part_len;
+        in += in_part_len;
     } while ((in_len -= limit) > 0);
 
     if (out_len_ptr)
-	*out_len_ptr = out_len;
+        *out_len_ptr = out_len;
```

Comment:
Please don't include irrelevant style-only changes. New code should use spaces instead of tabs, but existing code doesn't need to be replaced.

Reply:
Github does make reviewing this pretty easy if you use the ignore-whitespace mode: https://github.com/ruby/openssl/pull/359/files?w=1